### PR TITLE
[WIP] Experiments with containerd/runc (containerd v1.3.4 tag)

### PR DESCRIPTION
**Pull request by dims**
Signed-off-by: Davanum Srinivas <davanum@gmail.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cluster/gce/config-test.sh [hunk 1]
```diff
@@ -112,8 +112,8 @@ export LOAD_IMAGE_COMMAND
 # if KUBE_UBUNTU_INSTALL_CONTAINERD_VERSION or KUBE_UBUNTU_INSTALL_RUNC_VERSION
 # is set to empty then we do not override the version(s) and just
 # use whatever is in the default installation of containerd package
-export UBUNTU_INSTALL_CONTAINERD_VERSION=${KUBE_UBUNTU_INSTALL_CONTAINERD_VERSION:-}
-export UBUNTU_INSTALL_RUNC_VERSION=${KUBE_UBUNTU_INSTALL_RUNC_VERSION:-}
+export UBUNTU_INSTALL_CONTAINERD_VERSION="v1.3.4"
+export UBUNTU_INSTALL_RUNC_VERSION="v1.0.0-rc10"
```
Comment (bsdnet) [inline cluster/gce/config-test.sh:116]: What's the reason we hardcoded containerd and runc version?

Comment (BenTheElder) [reply cluster/gce/config-test.sh:116]: This PR is just using CI to test. See discussion in sig-testing on slack.

**Comment by BenTheElder** [comment]
@dims the install isn't overriding the host ...
https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/91733/pull-kubernetes-e2e-gce-ubuntu-containerd/1268333375463100416/artifacts/e2e-d104cf551b-a7d53-master/containerd.log

> time="2020-06-04T00:19:15.966643258Z" level=info msg="starting containerd" revision= version=1.3.3-0ubuntu2

https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/91733/pull-kubernetes-e2e-gce-ubuntu-containerd/1268333375463100416/artifacts/e2e-d104cf551b-a7d53-minion-group-f1jz/containerd.log

> time="2020-06-04T00:19:39.852238401Z" level=info msg="starting containerd" revision= version=1.3.3-0ubuntu2

**Comment by dims** [comment]
@BenTheElder that's the first time containerd starts. look at the log for when we restart it with the newer containerd.

time="2020-06-04T00:19:46.512106751Z" level=info msg="starting containerd" revision=814b7956fafc7a0980ea07e950f983d0837e5578 version=v1.3.4

from https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/91733/pull-kubernetes-e2e-gce-ubuntu-containerd/1268333375463100416/artifacts/e2e-d104cf551b-a7d53-minion-group-f1jz/containerd.log

**Comment by dims** [comment]
unrelated failure ...

/test pull-kubernetes-e2e-gce-ubuntu-containerd

**Comment by dims** [comment]
unrelated failure:

/test pull-kubernetes-e2e-gce-ubuntu-containerd

**Pull request closed without merging**